src/explorer/weights/WeightSlider.js

```diff
@@ -33,11 +38,19 @@ export class WeightSlider extends React.Component<Props> {
   }
 }
 
+function convertWeightToSliderValue(weight: number): number {
+  return weight === 0 ? WEIGHT_SLIDER_MIN : Math.log2(weight);
```

Comment:
Is it legal to call `convertWeightToSliderValue` with a negative value?
Currently, you return `NaN`.

Is it legal to call `convertWeightToSliderValue` with a value higher
than `2 ** WEIGHT_SLIDER_MAX`? Currently, you return a value outside of
the implied range.

Reply:
Added error handling and made all out-of-bounds inputs throw errors.